recipes/jupyterlab-fonts/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set version = "2.0.0" %}
+{% set name = "jupyterlab-fonts" %}
+{% set ns = "@deathbeds" %}
+{% set extensions = [
+  "jupyterlab-fonts",
+  "jupyterlab-font-anonymous-pro",
+  "jupyterlab-font-dejavu-sans-mono",
+  "jupyterlab-font-fira-code",
+] %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9c553ee9a1794859ba346e49473f1b260bbeba818e02f5651a9a91043ec0c912
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+    {% for ext in extensions %}
+    - cp {{ RECIPE_DIR }}/install.json {{ PREFIX }}/share/jupyter/labextensions/{{ ns }}/{{ ext }}/  # [unix]
+    - copy {{ RECIPE_DIR }}\install.json {{ PREFIX }}\share\jupyter\labextensions\{{ ns }}\{{ ext }}\  # [win]
+    {% endfor %}
+
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - jupyterlab ==3.*
```

You can simplify the operator
```suggestion
    - jupyterlab 3.*
```

---

Thanks, good point. Version specs are _still_ a dark art to me :sweat: !